presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
+* Remove SHOW STATS ON syntax
+* Restrict WHERE clause in SHOW STATS to filters that can be pushed down
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Immediately return final results to client for a failed query.
+
+Web UI
+---------------
+
+* Fix failure when rendering live plan view for queries involving index joins (issue #12054).
+
+Hive Changes
+---------------
+
+* Fix a corner case where ORC writer fails with "integer overflow" when writing
+  highly compressible data using dictionary encoding (#11930).
+* Queries reading Parquet files will now fail by default if statistics
+  in those Parquet files are corrupt (e.g., min > max).
+  To disable this behavior, set configuration property ``hive.parquet.fail-on-corrupted-statistics``
+  or session property ``parquet_fail_with_corrupted_statistics`` to false.
+
+Kudu Changes
+---------------
+
+* Update Kudu client to 1.8.0.
+* Add Kudu table property ``number_of_replicas`` to ``SHOW CREATE TABLE`` output.
```

Comment:
Do we have to say `Kudu` here given that this is a change for the Kudu connector?